client/extensions/woocommerce/components/store-deprecated-notice/style.scss

```diff
@@ -0,0 +1,8 @@
+.store-deprecated-notice {
+    .notice__icon-wrapper {
+        background: #d3af3f !important;
```

Comment:
I don't like using `!important`, usually it's possible to get the specificity right and you don't need it, but I guess because this won't stick around forever once store is removed this is ok.

Reply:
@becdetat I actually did not like it either. Let me see if I can get around it.

Reply:
@becdetat Added an additional class selector to increase specificity in https://github.com/Automattic/wp-calypso/pull/48436/commits/34eec1f668f1825bce4af6c1d8041fe6005a9d50